common/components/Header/components/GasPriceDropdown.jsx

```diff
@@ -0,0 +1,100 @@
+// @flow
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './GasPriceDropdown.scss';
+
+export default class GasPriceDropdown extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      expanded: false
+    };
+  }
+
+  static propTypes = {
+    gasPriceGwei: PropTypes.number,
+    gasPriceMinGwei: PropTypes.number,
+    gasPriceMaxGwei: PropTypes.number,
+    changeGasPrice: PropTypes.func
+  };
+
+  render() {
+    // col-xs-4 applies padding not observed in MEWv3, this overrides
+    const paddingReset = { paddingLeft: '0px', paddingRight: '0px' };
```

Comment:
Totally agree the scss file is underutilized at the moment. I'll move the styling over as suggested :smile: